canonical-web-and-design/vanilla-framework
# Remove custom icon styles from browser support page

## Done

Removed custom scss relating to icons on /browser-support and replaced the icons with img tags.

Fixes #982

## QA

- Open [demo](https://vanilla-framework-3631.demos.haus/browser-support)
- Check the icons at the bottom of the page, compare to [live site](https://vanillaframework.io/browser-support)
